src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/JsonRenderer/ListPackageJsonRenderer/ListPackageJsonOutputSerializerV1.cs

```diff
@@ -0,0 +1,365 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using NuGet.CommandLine.XPlat.Utility;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Protocol;
+using NuGet.Versioning;
+
+namespace NuGet.CommandLine.XPlat
+{
+    /// <summary>
+    /// Create json serialized output for string writer
+    /// </summary>
+    internal static class ListPackageJsonOutputSerializerV1
```

Comment:
Can all this be part of the renderer? 

Seems like the static class is not necessary. 

Did you want to have it separate because of the version? 
In that case, I'd argue that having the implementation be in one file is probably better anyways, as our versions are likely to be incremental.

Reply:
Sure, you're right. I overengineered it, merged into json renderer `ListPackageJsonRendererV1.cs`.